frontend/src/metabase/dashboard/components/DashCard.jsx

```diff
@@ -220,11 +223,8 @@ export default class DashCard extends Component {
           parameterValuesBySlug={parameterValuesBySlug}
           isEditing={isEditing}
           isPreviewing={this.state.isPreviewingCard}
-          gridSize={
-            this.props.isMobile
```

Comment:
We're using `gridSize` in a few other places -- `Funnel`, `Progress`, and `Text`. I haven't noticed any problems with always passing `gridSize`, but please LMK if you see anything.